Source/XCDBLD/BuildArguments.swift

```diff
@@ -6,10 +6,10 @@ public struct BuildArguments {
 	/// binary.
 	public enum BitcodeGenerationMode: String {
 		/// Only bitcode marker will be embedded.
-		case marker = "marker"
+		case marker
 
 		/// Full bitcode will be embedded.
-		case bitcode = "bitcode"
+		case bitcode
```

Comment:
I try to, but first I want to make sure you're aware of the community decisions. Since this is a PR about code style also, I think it makes sense to consider other opinions before just sticking to the status quo without asking. Since I added some changes due to personal preference here, too, I assume it's not always clear to you if this is a SwiftLint rule or one of mine / a custom rule. Thus I ask. I can understand that this can feel annoying, I'm sorry for that ...